color/src/color.rs

```diff
@@ -107,6 +107,46 @@ pub enum HueDirection {
     // NOTICE: If a new value is added, be sure to modify `MAX_VALUE` in the bytemuck impl.
 }
 
+/// Defines possible handling of alpha channel
+#[derive(Clone, Copy, Default, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[repr(u8)]
+#[expect(
+    clippy::exhaustive_enums,
+    reason = "There are only two ways to do interpolation."
+)]
+pub enum AlphaMode {
```

Comment:
Writing docs and naming was hard here. I also kept interpolation space explanation here.

Reply:
Actually we probably want to use this AlphaMode in other cases too, like https://github.com/linebender/color/pull/185#discussion_r2258468178.